tests/validation/tests/v3_api/test_airgap_ha.py

```diff
@@ -303,9 +352,6 @@ def setup_airgap_rancher(bastion_node, number_of_nodes=NUMBER_OF_INSTANCES):
     new_rancher_version = RANCHER_SERVER_VERSION
     if RANCHER_SERVER_VERSION[0] == 'v':
         new_rancher_version = RANCHER_SERVER_VERSION[1:]
-    extra_settings = []
-    if RANCHER_HELM_EXTRA_SETTINGS:
```

Comment:
Was this extra settings used before? If so why are you removing it now?

Reply:
there was an attempt at using it, but I tested it and it was not working properly. I tried to debug why, but decided to take it out until we actually need it.